lms/templates/courseware/course_updates.html

```diff
@@ -1,5 +1,6 @@
 <%! from django.utils.translation import ugettext as _ %>
-<section>
+<section aria-labelledby="course-updates-heading">
+    <h2 class="hd hd-2 sr" id="course-updates-heading">${_('All course updates')}</h2>
```

Comment:
Why an h2 instead of an h1? Note that this file isn't actually the course_updates HTML page we use in the LMS (https://clrux-ac-520.sandbox.edx.org/courses/course-v1:edX+DemoX+Demo_Course/info). I'm not sure where that template lives, but it uses an h1 instead of an h2.